netdata.spec.in

```diff
@@ -132,12 +134,14 @@ rm -rf $RPM_BUILD_ROOT
 %config(noreplace) %{_sysconfdir}/%{name}/*.conf
 %config(noreplace) %{_sysconfdir}/%{name}/python.d/*.conf
 %dir %{_sysconfdir}/%{name}
-%{?with_systemd:%{_unitdir}/netdata.service}
+%{_sysconfdir}/logrotate.d/%{name}
 %{_libexecdir}/%{name}
 %{_sbindir}/%{name}
 %dir %{_datadir}/%{name}
+%dir %{_sharedstatedir}/%{name}
```

Comment:
are you sure you want the dir to be owned root:root 755?
i think it should be `%attr(755,netdata,netdata)` at least!